interbotix_xs_toolbox/interbotix_xs_modules/interbotix_xs_modules/xs_robot/arm.py

```diff
@@ -117,6 +119,12 @@ def __init__(
             object is built; set to `False` if intending to sub-class this. If set to `False`,
             either call the `start()` method later on, or add the core to an executor in another
             thread.
+        :param node_owner: (Used when controlling multiple robots) set to `True` to give access
+            to rclpy states like init and shutdown. For multiple robots, only one robot should
+            get the node ownership. Every other robot will use the node but cannot control init
+            or shutdown.
+        :param update_kinematics: (optional) decides whether or not to update forward
```

Comment:
Add a note to this param stating that users can call the arm module's `capture_joint_positions` or `_update_Tsb` directly if they want to update the arm module's FK.